styles.css

```diff
@@ -15,6 +15,8 @@ html {
 body {
   font-family: 'Poppins', sans-serif;
   background: #f7f7f9;
+
+  /* overflow-x: hidden; */
```

Comment:
- [ ] Kindly consider removing the commented block of code in your codebase as it's not a good practice for clean code.